src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/DynamicRevocationTests.Android.cs

```diff
@@ -11,6 +11,6 @@ namespace System.Security.Cryptography.X509Certificates.Tests.RevocationTests
 {
     public static partial class DynamicRevocationTests
     {
-        public static bool SupportsDynamicRevocation { get; } = Interop.AndroidCrypto.X509ChainSupportsRevocationOptions();
```

Comment:
Is X509ChainSupportsRevocationOptions used anywhere else, or can it now be deleted?

Reply:
Looks like it's not used in the managed layer, only in native.